packages/os/autoscaling-warm-pool.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Run tasks reliant on IMDS
+After=network-online.target settings-applier.service
+Requires=settings-applier.service
+Wanted=network-online.target
+Before=kubelet.service
+# We only want to run once, at first boot. This file is created by shibaken
+# after a successful run.
+ConditionPathExists=!/var/lib/bottlerocket/shibaken.ran
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/shibaken boot-shibaken
```

Comment:
I'm not sure that we want to have `autoscaling-warm-pool` as the name of the service running `shibaken`'s `boot-shibaken` service mode. There should be a parent `shibaken` service instead, or the ExecStart should be something like `/usr/bin/shibaken is_warmpool_healthy`.

Reply:
I'd expect something like:
* unit name - `wait-for-in-service.service`
* unit description - "Wait for the instance to be in service"
* `shibaken` subcommand - `shibaken wait-for-in-service`